lte/gateway/python/integ_tests/s1aptests/s1ap_utils.py

```diff
@@ -1322,3 +1325,53 @@ def offload_agw(self, imsi, enbID, offloadtype=0):
             self._ha_stub.StartAgwOffload(req)
         except grpc.RpcError as e:
             print("gRPC failed with %s: %s" % (e.code(), e.details()))
+
+
+class HeaderEnrichmentUtils:
+    def __init__(self):
+        self.magma_utils = MagmadUtil(None)
+        self.dump = None
+
+    def restart_envoy_service(self):
+        print("restarting envoy")
+        self.magma_utils.exec_command_output("sudo service magma@envoy_controller restart")
+        time.sleep(5)
+        self.magma_utils.exec_command_output("sudo service magma@envoy restart")
+        time.sleep(30)
+        print("restarting envoy done")
+
+    def get_envoy_config(self):
+        output = self.magma_utils.exec_command_output(
+            "sudo ip netns exec envoy_ns1 curl 127.0.0.1:9000/config_dump")
+        self.dump = json.loads(output)
+
+        return self.dump
+
+    def get_route_config(self):
+        self.dump = self.get_envoy_config()
+
+        for conf in self.dump['configs']:
+            if 'static_route_configs' in conf:
+                return conf['static_route_configs']
```

Comment:
```
        for conf in self.dump['configs']:
            if 'dynamic_listeners' in conf:
                return conf['dynamic_listeners'][0]['active_state']['listener']['filter_chains'][0]['filters']
```

Reply:
I see, I did not notice the 'dynamic_listener' section. I think that is right config to set.
But we do need to check why 'static_coute_config' is set in envoy-config.